packages/frontend/amp/components/elements/VideoYoutubeBlockComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export const VideoYoutubeBlockComponent: React.FC<{
+    element: VideoYoutubeBlockElement;
+}> = ({ element }) => {
+    const youtubeId = element.url.split('v=').pop();
```

Comment:
This kind of thing makes me nervous - although I'm guessing it's just what we do already. Is it worth performing at least some kind of check on the value, or is that overkill?

Reply:
Well.. I feel like I may have totally over-engineered it now but I've made a validator for getting IDs from URLs, if you could take a look!

Reply:
Ha looks good! My only question is whether the Youtube matching is a bit brittle (if they change their ID format?) but I think either way is fine - possibly with logging something in the error case?

Reply:
Yeah, I thought the same - this is a cut down version of the regex in [existing AMP](https://github.com/guardian/frontend/blob/7015e549078345047d80ff1925b2782b499972f3/common/app/views/support/cleaner/AmpEmbedCleaner.scala#L67) but also.. my thoughts are if they change their format, they're gonna break *a lot* of things and I'm happy here that it will log the error and we can catch it fairly widely. 

I'll add a standard string for the log error though so we can track in Kibana!